src/server.rs

```diff
@@ -607,9 +637,7 @@ impl LanguageServer for LspServer {
                                         continue;
                                     }
                                     let location =
-                                        convert::node_to_location(
-                                            &node, key,
-                                        );
+                                        node_to_location(&node, key);
```

Comment:
Yeah, same here: changing the import path here doesn't buy us anything, but now the patch is a little noisier than we want.